client-network/src/network_ops.rs

```diff
@@ -15,6 +17,15 @@ use secstr::SecUtf8;
 
 /// Interface for performing network operations on Crypto.com Chain
 pub trait NetworkOpsClient {
+    /// fetch current StakedState
+    fn get_staked_state_account(
+        &self,
+        to_staked_account: StakedStateAddress,
+    ) -> Result<StakedState>;
+
+    /// fetch current StakedState nonce
+    fn get_staked_state_nonce(&self, to_staked_account: StakedStateAddress) -> Result<Nonce>;
```

Comment:
I don't think this function adds any value. `get_staked_state_account()` already returns `StakedState` object which contains nonce along with other details.

Reply:
get_staked_state_nonce is for convenience.
what do you think?

Reply:
If this is a convenience method, I don't think it belongs in the interface. Also, I think below code is perfectly reasonable.
```
let nonce = client.get_staked_state_account()?.nonce;
```
So, I don't think we're adding any value by creating a new function just to avoid accessing field using `.nonce`.

Reply:
got it